### DISCUSSION TITLE
addon 支持对特定集群扩容

### DISCUSSION THREAD

**Discussion by ThuWangzw** [kubevela/kubevela#5355] [Ideas]
目前，addon enable 时不支持不同集群使用不同的 parameter，所有修改都应该通过[修改 cue template](https://kubevela.net/docs/case-studies/multi-cluster#override-default-configurations-in-clusters) 的方式进行。

然而，当 addon 在某个集群占用的资源增加，需要扩容时，通过上述方式来扩容会非常麻烦，求一个更简单的方式。

推广到更 general 的情况，能否支持对不同集群使用不同的 parameter？

**Comment by wonderflow** [comment]
你期望的方式是怎么样的？

**Reply by ThuWangzw** [reply]
我期望能通过 cli 对集群的参数进行调整，例如在 vela addon enable 时对不同集群有不同的 parameter，或者添加一个 vela addon edit 命令？
